src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/AimdRateControl.java

```diff
@@ -456,13 +460,15 @@ public long updateBandwidthEstimate(long nowMs)
                     currentInput.incomingBitRate,
                     nowMs);
 
-        if (isValidEstimate() && logger.isTraceEnabled())
+        DiagnosticContext diagnosticContext = this.diagnosticContextProvider.getDiagnosticContext();
+        if (isValidEstimate()
```

Comment:
Not sure how expensive isValidEstimate is, but I'd move isTraceEnabled before it since that is probably more likely to return false